client/src/pages/Houses/styled.js

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components'
+
+const alignCenter = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`
+
+export const HousesContainer = styled.div`
+  height: 90vh;
+  ${alignCenter};
+  justify-content: flex-start;
+`
+
+export const HousesComponentContainer = styled.div`
+  margin-top: 20px;
+  ${alignCenter}
+`
+
+export const Img = styled.img`
+  margin-top: 20px;
```

Comment:
img tags that need just a little bit of extra styling and can expect to always render the same image source can have the img src attached via styled props attrs
See https://styled-components.com/docs/api#attrs

For example 
```js
import confusedPath from '../../images/confused.svg'


export const Img = styled.img.attrs({
    src: `${confusedPath}`
    alt="doesn't exist"
})`
  margin-top: 20px;
`

```

Reply:
This is relevant in this example, because you're importing the img path in the Houses Component and passing it to the `src` prop of your Img styled component. But since that img path is static, it's not really necessary to pass it as a prop. Instead, you can set it directly here.